### PR TITLE
Add issue tracking action for the extension

### LINKED ISSUE #932: Add a command to report issues on github
Add `Report an Issue` command to view's command bar:

image

### PR DESCRIPTION
**Pull request by mohitsuman**
Fix #932 

1.  This also enhances the following icons in consistency with dark and light themes.

- Login
- Refresh

2. The below screenshot shows the UI representation:

Screenshot 2019-07-16 at 4 19 23 AM

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -157,6 +158,23 @@ async function isOpenShift(): Promise<boolean> {
...
+        'OS:': Platform.OS,
+        'Extension version:': vscode.extensions.getExtension('redhat.vscode-openshift-connector').packageJSON.version
+    };
+    for (const [key, value] of Object.entries(template)) {
+        body = `${body}${key} ${value}\n`;
+    }
+
+    return vscode.commands.executeCommand(
+        'vscode.open',
+        vscode.Uri.parse(`${repoURL}/issues/new?labels=bug&title=Issue&body=**Environment**\n${body}\n**Description**`));
```
Comment (dgolovin) [inline src/extension.ts]: labels should be ```labels=kind%2Fbug```

Comment (mohitsuman) [reply src/extension.ts]: Fixed

**Comment by jrichter1** [comment]
About the CI timeouts, I'm guessing it's due to the openshift.issueTracker command being called here:
https://github.com/redhat-developer/vscode-openshift-tools/blob/master/test/extension.test.ts#L73
That opens up a browser and leaves is hanging. 

The easiest fix I can think of would be to exclude it from the filter here:
https://github.com/redhat-developer/vscode-openshift-tools/blob/master/test/extension.test.ts#L64

**Pull request merged**